extension/intents/pocket/pocket.js

```diff
@@ -0,0 +1,27 @@
+import * as pageMetadata from "../../background/pageMetadata.js";
+import * as intentRunner from "../../background/intentRunner.js";
+import * as content from "../../background/content.js";
+
+intentRunner.registerIntent({
+  name: "pocket.open",
+  async run(context) {
+    const url = "https://app.getpocket.com/";
+    await context.createTab({ url });
```

Comment:
This could use `context.openOrFocusTab()` so it focuses an existing Pocket tab if it exists.